Objects/listobject.c

```diff
@@ -571,18 +571,20 @@ list_repeat(PyListObject *a, Py_ssize_t n)
     }
     else {
         PyObject **src = a->ob_item;
-        PyObject **src_end = src + Py_SIZE(a);
         while (src < src_end) {
-            Py_SET_REFCNT(*src, Py_REFCNT(*src) + n);
+            Py_SET_REFCNT((elem = src[i]), Py_REFCNT(elem) + n);
```

Comment:
Again, this is unspecified behavior you can't rely on. Don't assign to `elem` in the one argument where the other argument is using it.

Reply:
Where is `i` even coming from? I don't see it defined. Is there a benefit to replacing the pointer arithmetic with indexing? Changing it just for personal preference, not as a necessary part of the patch, is frowned upon.

Reply:
It's not "just for personal preference", it's used later to assign to `dest` without having to modify it.

Reply:
> I know that, but I usually can't verify it because I make changes on the desktop website (I don't know how to use `git`).

You're going to have to learn. It's literally the first thing you do to begin contributing. See the [Python Developer's Guide: Setup](https://devguide.python.org/setup/).